src/legend_data_monitor/__main__.py

```diff
@@ -1,6 +1,6 @@
 """legend_data_monitor.__main__: executed when legend_data_monitor directory is called as script."""
 
 
-from .legend_data_monitor import main
+from . import main
 
 main()
```

Comment:
Per creare eseguibili vi consiglio di dichiararli in `setup.cfg`. C'è già qualcosa impostato come `console_script` ma non mi sembra funzionante...